### PR TITLE
New package: mlt-0.9.6

### PR DESCRIPTION
**Pull request by diogoleal**

### CODE REVIEW

**Comment by xtraeme** [comment]
- dependencies are wrong for cross compilation
- this needs a development subpkg

**Comment by diogoleal** [comment]
@xtraeme Sorry, but I could not understand about "dependencies are wrong for cross compilation."
Its you give me an example?

**Comment by thypon** [comment]
the -devel packages should stay in makedepends while the binaries executed during compilation phase should stay in hostmakedepends.

File: srcpkgs/mlt/template [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Template file for 'mlt'
+pkgname=mlt
+version=0.9.6
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr ${XBPS_MAKEJOBS:+--parallel=$XBPS_MAKEJOBS}"
```
Comment (xtraeme) [inline srcpkgs/mlt/template]: This `configure_args` line is completely unnecessary.

**Comment by pullmoll** [comment]
Please rebase and fix the `common/shlibs` addition to not include unrelated lines.

**Comment by pullmoll** [comment]
@diogoleal I took your package and added missing bits and pieces, such as all dependency packages we have in Void.

I'm not sure if the `ladspa-sdk-example-plugins` is all that useful, but without it `melt` complains it can't find any plugins.

**Pull request closed without merging**